pkg/controller/cronjob/cronjob_controller.go

```diff
@@ -285,7 +285,7 @@ func syncOne(cj *batchv1.CronJob, js []batchv1.Job, now time.Time, jc jobControl
 	}
 	if tooLate {
 		klog.V(4).Infof("Missed starting window for %s", nameForLog)
-		recorder.Eventf(cj, v1.EventTypeWarning, "MissSchedule", "Missed scheduled time to start a job: %s", scheduledTime.Format(time.RFC1123Z))
+		recorder.Eventf(cj, v1.EventTypeWarning, "MissSchedule", "Missed scheduled time to start a job: %s", scheduledTime.UTC().Format(time.RFC1123Z))
```

Comment:
where does the timezone in `scheduledTime` come from? is it the controller time zone?

Reply:
https://github.com/kubernetes/kubernetes/blob/ea0764452222146c47ec826977f49d7001b0ea8c/pkg/controller/cronjob/cronjob_controller.go#L137

Yes, its timezone comes from the controller, and when we don't use the timezone explicitly, the timezone of `time.Now()` will be set to nil, which will make `time.Format()` use the controller's timezone

Reply:
would like an ack from a sig-apps review
/assign @soltysh

Reply:
Yeah, this should be fine.